hassio/src/components/hassio-progress.ts

```diff
@@ -0,0 +1,83 @@
+import { UnsubscribeFunc } from "home-assistant-js-websocket";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { SubscribeMixin } from "../../../src/mixins/subscribe-mixin";
+import { HomeAssistant } from "../../../src/types";
+import "@polymer/paper-progress/paper-progress";
```

Comment:
Can we use [`mwc-linear-progress`](https://github.com/material-components/material-components-web-components/tree/master/packages/linear-progress)?

Reply:
YAS. I actually came to this PR to say this :)

Reply:
Pls take a look